src/features/More/moreApi.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+import Strings from '../../constants/Strings';
+
+const moreApi = {
+  getCategories: async () => {
+    try {
+      const result = await axios.get(
+        `${Strings.moreScreen.api.BASE_URL}${Strings.moreScreen.api.getAllCategories.ENDPOINT}`,
```

Comment:
The syntax is good but is the base URL belong to the more screen?
Endpoints should be stored in a separate section